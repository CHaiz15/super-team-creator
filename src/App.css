@import url('https://fonts.googleapis.com/css?family=Bangers&display=swap');

.App {
  text-align: center;
  font-family: 'Bangers';
}

