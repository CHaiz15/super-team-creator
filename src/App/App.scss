@import url('https://fonts.googleapis.com/css?family=Bangers&display=swap');

body {
  font-family: 'Bangers';
  background: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), url('../Assets/main-bg.jpeg');
  height: 100vh;
}
