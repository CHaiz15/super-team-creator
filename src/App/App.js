import React from 'react'
import './App.scss'
import { Route } from 'react-router-dom'
import Home from '../Home/Home.js'
import CreationZone from '../CreationZone/CreationZone'
import TeamDisplay from '../TeamDisplay/TeamDisplay'
import Favorites from '../Favorites/Favorites'
import FavTeamDisplay from '../FavTeamDisplay/FavTeamDisplay'

export const App = () => {
  return (        
    <>
      <Route exact path='/' component={Home} />
      <Route exact path='/create' component={CreationZone} />
      <Route exact path='/team' component={TeamDisplay} />
      <Route exact path='/favorites' component={Favorites} />
      <Route exact path='/favTeam' component={FavTeamDisplay} />
    </>
  );
}

export default App;
