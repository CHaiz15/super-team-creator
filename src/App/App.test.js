import React from 'react';
import App from './App';
import { shallow } from 'enzyme'

describe('App', () => {
  it('should be an instance of the App component', () => {
    const wrapper = shallow(<App/>);
      expect(wrapper).toMatchSnapshot();
  });
})
