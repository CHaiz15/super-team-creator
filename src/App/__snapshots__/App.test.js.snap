// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should be an instance of the App component 1`] = `ShallowWrapper {}`;
